src/assets/js/modal.js

```diff
@@ -129,7 +129,9 @@ export default class Modal {
                 sessionStorage.setItem('username', yourName);
                 //create room link
                 let roomgen = lib.roomgen(roomName,med.h.generateRandomString())
-                let roomLink = lib.roomLink(location.origin,roomgen) 
+                // save room name in sessionStorage
```

Comment:
Saving it to the session is one way to solve it. But I thought when people logout using the red button, it should maybe just go to the room url again this way we wont loose the room or is this assumption incorrect?